config/500-controller.yaml

```diff
@@ -60,7 +60,7 @@ spec:
         - name: CONFIG_OBSERVABILITY_NAME
           value: config-observability
         - name: METRICS_DOMAIN
-          value: cloud.run/events
+          value: knative.dev/eventing
```

Comment:
This seems wrong

Reply:
In order to register monitored resources with stackdriver, we need to specify the metrics domain. We are proposing three new resources: knative_broker, knative_trigger, knative_importer, which have metrics in certain domain. E.g.,  
`knative.dev/eventing/importer/event_count `... 
`knative.dev/eventing/importer/dispatch_latencies`, ...

I agree it would be nice to keep the cloud.run/events, but I think this is the generic way allowing importers from different parties to leverage SD monitored resources. Otherwise, they will need to go over a SD approval process to add new metrics to those resources for their own domains... Does it make sense?